resolwe_bio/processes/reads_processing/trimmomatic.yml

```diff
@@ -0,0 +1,415 @@
+---
+
+- slug: trimmomatic-single
+  name: trimmomatic (single-end)
+  data_name: '{{ reads.fastq.0.file|default:"?" }}'
+  version: 0.0.1
+  type: data:reads:fastq:single:trimmomatic
+  category: analyses
+  flow_collection: sample
```

Comment:
Why not required? Shouldn't the filtered reads be part of the sample. 


> On Aug 29, 2016, at 9:01 PM, jkokosar <notifications@github.com> wrote:
> 
> In resolwe_bio/processes/reads_processing/trimmomatic.yml <https://github.com/genialis/resolwe-bio/pull/101#discussion_r76720251>:
> 
> > @@ -0,0 +1,415 @@
> > +---
> > +
> > +- slug: trimmomatic-single
> > +  name: trimmomatic (single-end)
> > +  data_name: '{{ reads.fastq.0.file|default:"?" }}'
> > +  version: 0.0.1
> > +  type: data:reads:fastq:single:trimmomatic
> > +  category: analyses
> > +  flow_collection: sample
> This line is not required and will also be removed from other reads-filtering processors.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/genialis/resolwe-bio/pull/101/files/2559e6ef9b10f4f79bd8edecf3a07e3dec29772e#r76720251>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AA-qnJUt8qglZKvMqC6ZBxbeD-WKdDiqks5qk474gaJpZM4Jsz_D>.
>